release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -35,6 +35,8 @@ Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to
 //TODO: The line below should be used when it is next appropriate. Revisit in August 2023 timeframe.
 Maintenance support ends for version 4.12 on 25 January 2025 and goes to extended life phase. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
 
+Commencing with the {product-version} release, Red Hat is simplifying the administration and management of Red Hat shipped Cluster Operators with the introduction of 3 new life cycle classifications; Platform Aligned, Platform Agnostic and Rolling Stream. These life cycle classifications will provide additional ease and transparency for Cluster administrators to understand the life cycle policies of each Operator, and form cluster maintenance and upgrade plans with predictable support boundaries. For more information, see link:https://access.redhat.com/webassets/avalon/j/includes/session/scribe/?redirectTo=https%3A%2F%2Faccess.redhat.com%2Fsupport%2Fpolicy%2Fupdates%2Fopenshift_operators[OpenShift Operator Life Cycles].
```

Comment:
```suggestion
Commencing with the {product-version} release, Red Hat is simplifying the administration and management of Red Hat shipped cluster Operators with the introduction of three new life cycle classifications; Platform Aligned, Platform Agnostic, and Rolling Stream. These life cycle classifications provide additional ease and transparency for cluster administrators to understand the life cycle policies of each Operator and form cluster maintenance and upgrade plans with predictable support boundaries. For more information, see link:https://access.redhat.com/webassets/avalon/j/includes/session/scribe/?redirectTo=https%3A%2F%2Faccess.redhat.com%2Fsupport%2Fpolicy%2Fupdates%2Fopenshift_operators[OpenShift Operator Life Cycles].
```
Otherwise, LGTM